src/Agent/NewRelic/Agent/Core/Configuration/ServerConfiguration.cs

```diff
@@ -326,4 +329,15 @@ public void OnError(StreamingContext context, ErrorContext errorContext)
             Log.ErrorFormat("Json serializer context path: {0}. Error message: {1}", errorContext.Path, errorContext.Error.Message);
         }
     }
+
+    [JsonConverter(typeof(ExpectedStatusCodesConverter))]
+    public class ExpectedStatusCodes
+    {
+        public ExpectedStatusCodes(IEnumerable<string> expectedStatusCodesArray)
+        {
+            ExpectedStatusCodesArray = expectedStatusCodesArray;
+        }
+
+        public IEnumerable<string> ExpectedStatusCodesArray { get; private set; }
+    }
```

Comment:
This feels unnecessary.  It seems that the only use case is an string[] which would make this class unnecessary.